Framework/Source/Graphics/RenderGraph/RenderGraph.cpp

```diff
@@ -258,6 +261,56 @@ namespace Falcor
 
     bool RenderGraph::isValid(std::string& log) const
     {
+        std::unordered_map<RenderPass*, RenderPassReflection> passReflectionMap;
```

Comment:
Did that code come from @kyaoNV ?

Reply:
no it's based on his generate edges.

Reply:
@kyaoNV This code should exist in some form in your branch and I did not review it